get_version_git() uses git describe which describes the aports repo, it is supposed to fail and use get_version() instead

diff --git a/contrib/get-version.py b/contrib/get-version.py
index 0a678ca..f6fe0d7 100755
--- a/contrib/get-version.py
+++ b/contrib/get-version.py
@@ -39,9 +39,4 @@ def get_version():
 
 if __name__ == '__main__':
 
-    version = get_version_git()
-    if version:
-        version = sanitize_for_ci(version)
-    else:
-        version = get_version()
-    print(version)
+    print(get_version())
